dev/ci/user-overlays/10446-ejgallego-proof+remove_sign.sh

```diff
@@ -0,0 +1,13 @@
+if [ "$CI_PULL_REQUEST" = "10446" ] || [ "$CI_BRANCH" = "proof+remove_sign" ]; then
+
+    true
+    # bignums_CI_REF=proof+remove_sign
+    # bignums_CI_GITURL=https://github.com/ejgallego/bignums
+
+    # compcert_CI_REF=proof+remove_sign
+    # compcert_CI_GITURL=https://github.com/ejgallego/CompCert
+
+    # vst_CI_REF=proof+remove_sign
+    # vst_CI_GITURL=https://github.com/ejgallego/VST
+
+fi
```

Comment:
Please remove this file.

Reply:
I left it for "documentation purposes", removed now.